pages/blog/sandbox.tsx

```diff
@@ -14,6 +14,28 @@ interface Props {
 }
 export default function Sandbox({ frontMatter, source }: Props): JSX.Element {
 	const router = useRouter();
+	const [isScrollable, setScrollable] = useState(false);
+
+	// Scroll to top on button click
+	const handleScrollToTop = (): void => {
+		window.scrollTo({ top: 0, behavior: 'smooth' });
+	};
+
+	if (typeof window !== 'undefined') {
+		window.onscroll = (): void => {
+			makeButtonVisible();
+		};
+	}
+
+	// Appear scrollable when the user scrolls down
+	const makeButtonVisible = (): void => {
+		const scrollTop = window.pageYOffset;
+		if (scrollTop > 0) {
+			setScrollable(true);
+		} else {
+			setScrollable(false);
+		}
+	};
```

Comment:
Let's use the [useScroll()](https://github.com/streamich/react-use/blob/master/docs/useScroll.md) hook provided to us by the `react-use` library. Another simpler [implementation](https://github.com/blaccsmith/website/blob/c3d4c3e437cb35fa6e6a441a4125ed3ea245299e/pages/blog/%5Bslug%5D.tsx#L19-L25) used in a different branch

Reply:
Updated with the implementation used in a different branch